examples/counter/shared/src/capabilities/sse.rs

```diff
@@ -35,7 +35,7 @@ where
 
     pub fn get_json<F, T>(&self, url: impl AsRef<str>, make_event: F)
     where
-        F: Fn(T) -> Ev + Clone + Send + 'static,
+        F: FnOnce(T) -> Ev + Clone + Send + 'static,
```

Comment:
I think this needs to stay as `Fn(T)` because we need to call it repeatedly for every chunk received from the server.

Reply:
I was looking at that, but we seem to clone it for every call...?

Reply:
for every call, but not for every chunk in the response

Reply:
Ah yes, I see it now